Day-6/src/components/GithubCard.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from "react";
+
+const GithubCard = ({ data }) => {
+  const [teamData, setTeamData] = useState([]);
+  console.log(data);
+
+  useEffect(() => {
+    displayUserData();
+  }, []);
+
+  const fetchUserData = async (url) => {
+    const response = await fetch(url);
+    const data = response.json();
+    return data;
+  };
+
+  const displayUserData = () => {
+    const output = [];
+    data &&
```

Comment:
I was also thinking of the same workaround, but the solution is here in your code. You did API calls in a loop and added data to the state. Appreciated.

Reply:
yeah!